modules/core/src/lib/deck.js

```diff
@@ -250,9 +254,15 @@ export default class Deck {
 
     const viewManagerNeedsRedraw = this.viewManager.needsRedraw({clearRedrawFlags});
     const layerManagerNeedsRedraw = this.layerManager.needsRedraw({clearRedrawFlags});
+    const effectManagerNeedsRedraw = this.effectManager.needsRedraw({clearRedrawFlags});
```

Comment:
Unrelated: animation loop can also need redraw... @tsherif

Reply:
This recurssively getting redraw flag thing is getting quite heavy. We might want to consider moving it to a push model.